metrics/api/v1/model_handlers.go

```diff
@@ -33,7 +33,26 @@ import (
 // when manager.model has not been initialized.
 var errModelNotActivated = errors.New("the model is not activated")
 
-var metricNamesConversion = map[string]string{
+var supportedMetrics = map[string]struct{}{
```

Comment:
Or let's do it other way round, I will merge and apply a quick fix. I need enable custom metrics ASAP and the current code block them.